engine/runtime/src/main/java/org/enso/ClassLoaderConstants.java

```diff
@@ -0,0 +1,11 @@
+package org.enso;
+
+import java.util.List;
+
+public class ClassLoaderConstants {
+
+  public static final List<String> RESOURCE_DELEGATION_PATTERNS = List.of("org.slf4j", "ch.qos");
+  public static final List<String> CLASS_DELEGATION_PATTERNS =
+      List.of("org.graalvm", "java", "org.slf4j", "ch.qos");
```

Comment:
It would be good to add some doc explaining what these patterns are for.

e.g. I assume these are patterns that determine packages which are always loaded using the system class loader? A comment saying that would make it easier to understand.

Reply:
Docs added in cb6c6a18c592e27d1d67676d75ce807c0b023321